src/css/index.css

```diff
@@ -0,0 +1,526 @@
+}
+
+.header__title-main {
+    font-size: var(--h1-font);
+    color: var(--main-text-color);
+}
+
+.header__title-sub {
+    font-size: var(--h4-font);
+    color: var(--sub-text-color);
+}
+
+.header__panel {
+    display: flex;
+    margin: 0;
+    padding: 0;
+    list-style-type: none;
+    font-size: var(--p-link-font);
+}
+
+.header__panel-link {
+    display: block;
+    border-radius: 2px;
+    padding: 14px 16px;
+    text-align: center;
+    text-decoration: none;
+    color: var(--sub-text-color);
+}
+
+.header__panel-link:hover {
+    background-color: var(--button-bg-color);
+    color: var(--button-fg-color);
+}
+
+/*##banner*/
+
+.banner__content-container {
+    display: flex;
+    flex-wrap: wrap;
+    height: 440px;
+    background-image: url("../assets/banner.png");
+}
+
+.banner__content-item {
+    display: flex;
+    align-items: flex-end;
+    flex-basis: 50%;
+    color: var(--banner-item-color);
+}
+
+.banner__content-background {
+    width: 100%;
+    height: 100%;
+    background-color: var(--banner-bg-color);
+}
+
+.banner__content-item:nth-child(1) .banner__content-background {
+    opacity: 0.6;
+}
+
+.banner__content-item:nth-child(2) .banner__content-background {
+    opacity: 0.4;
+}
+
+.banner__content-item:nth-child(3) .banner__content-background {
+    opacity: 0.4;
+}
+
+.banner__content-item:nth-child(4) .banner__content-background {
+    opacity: 0.1;
+}
+
+.banner__content-item:nth-child(4) .banner__content-text {
+    background-color: var(--button-bg-color);
+    color: white;
+}
+
+.banner__content-text {
+    position: absolute;
+    padding: 10px;
+    background-color: var(--banner-bg-color);
+    font-size: var(--h3-font);
+}
+
+/*##title*/
+
+.title {
+    margin-top: var(--default-block-margin);
+    border-bottom: 1px solid var(--title-border-color);
+}
+
+.title__content {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: space-between;
+    margin-bottom: var(--default-block-margin);
+}
+
+.title__content-text {
+    flex-basis: 75%;
+}
+
+.title__content-main-text {
+    font-size: var(--h1-font);
+    color: var(--main-text-color);
+}
+
+.title__content-subtext {
+    font-size: var(--h3-font);
+    color: var(--sub-text-color);
+}
+
+.title__content-img {
+    font-size: var(--h3-font);
+    color: var(--button-fg-color);
+}
+
+/*##row*/
+
+.row {
+    margin-top: var(--default-block-margin);
+}
+
+.row__content {
+    display: flex;
+    flex-flow: row;
+    flex-wrap: wrap;
+    margin-top: 10px;
+    width: 100%;
+}
+
+.row__content-item {
+    display: flex;
+    justify-content: left;
+    align-items: center;
+    width: 50%;
+    margin: 10px auto;
+}
+
+/*rounded images*/
+.row__content-span {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    border: 2px solid var(--row-circle-color);
+    border-radius: 130px;
+    min-height: 80px;
+    min-width: 80px;
+    width: 8vw;
+    height: 8vw;
```

Comment:
- if you are using it for span then width/hight will not work (unless you set up the display as you did).
- don't name the class according to the tag you are supposing to use it - it will make you feel that the class useless for other tags(but will work for them the same way).
- check out the units.